main.tf

```diff
@@ -3,32 +3,41 @@
 
 module "vcn" {
   source  = "oracle-terraform-modules/vcn/oci"
-  version = "3.0.0-RC1"
+  version = "3.0.0"
 
+  # general oci parameters
   compartment_id = var.compartment_id
   label_prefix   = var.label_prefix
 
   # gateways
-  create_drg                   = var.create_drg
-  drg_display_name             = var.drg_display_name
-  internet_gateway_enabled     = true
-  local_peering_gateways       = var.local_peering_gateways
-  lockdown_default_seclist     = var.lockdown_default_seclist
-  nat_gateway_enabled          = var.worker_mode == "private" || var.create_operator == true || (var.lb_type == "internal" || var.lb_type == "both") ? true : false
-  nat_gateway_public_ip_id     = var.nat_gateway_public_ip_id
-  service_gateway_enabled      = true
-  tags                         = var.tags["vcn"]
-  vcn_cidr                     = var.vcn_cidr
+  create_internet_gateway  = true
+  create_nat_gateway       = var.worker_mode == "private" || var.create_operator == true || (var.lb_type == "internal" || var.lb_type == "both") ? true : false
+  create_service_gateway   = true
+  nat_gateway_public_ip_id = var.nat_gateway_public_ip_id
+
+  # drg
+  create_drg       = var.create_drg
+  drg_display_name = var.drg_display_name
+
+  # lpgs
+  local_peering_gateways = var.local_peering_gateways
+
+  # tags
+  freeform_tags = var.tags["vcn"]
```

Comment:
lets keep the freeform_tags naming convention across the modules.

Reply:
This PR is for VCN 3.0.0. Can you please open a new issue for the freeform_tags?